nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestAutomaticIndexing.java

```diff
@@ -176,6 +177,35 @@ public void setupIndex() throws Exception {
         esa.initIndexes(true);
     }
 
+//    @Test
+    public void shouldNotIndexRootDocument() throws Exception {
+        startTransaction();
+        // update acp on root document as it is reinit during tear down
```

Comment:
could be interesting to add some assertion that root is never indexed
- after a framework init
- after a reindex all
- after a reindex with nxql using the root uid